slave-base/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:centos7
+
+MAINTAINER Ben Parees <bparees@redhat.com>
+
+ENV HOME=/home/jenkins
+
+USER root
+# Install headless Java
+RUN yum install -y --setopt=tsflags=nodocs --enablerepo=centosplus epel-release && \
+    rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7 && \
+    INSTALL_PKGS="bc gettext git java-1.8.0-openjdk-headless lsof nss_wrapper rsync tar unzip which zip" && \
+    yum install -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all && \
+    mkdir -p /home/jenkins && \
+    chown -R 1001:0 /home/jenkins && \
+    chmod -R g+w /home/jenkins
+# Copy the entrypoint
+ADD contrib/bin/* /usr/local/bin/
+
+# Run the JNLP client by default
+# To use swarm client, specify "/opt/app-root/jenkins/run-swarm-client" as Command
```

Comment:
Is swarm already getting installed somewhere else (if so, I'm not seeing it)?  If not, should we clarify that somehow (or is it too obvious)?